README.md

```diff
@@ -102,13 +81,6 @@ High level `make` targets:
 * `system-test`: run the networking/"sanity" tests. Specify
   `CONTIV_NODE_OS=centos` to test on centos instead of ubuntu.
 
-#### Resource Allocation
```

Comment:
ah nice, I didn't know that.

@jainvipin do you want to do that as part of this PR i.e. move CONTRIBUTING.md out of docs?